### ISSUE TITLE
25 Closures -- first example doesn't output what it says

### ISSUE THREAD

**Issue by wjl** [munificent/craftinginterpreters#702]
The first example at the beginning of the chapter is this:

```
var x = "global";
fun outer() {
  var x = "outer";

  fun inner() {
    print x;
  }

  inner();
}
```

Then the text says:

> Run this example now and it prints “global”. It’s supposed to print “outer”. 

However, the example doesn't actually print **anything**, because it just defines the function, but doesn't actually call it.

Seems like you should either:
- add a call to outer() in the code, or
- reword the text to say "calling outer() now prints" or something like that.

**Comment by wjl** [comment]
By the way, I've found a few more between this and chapter 28 where the code just declares a function, class, or method, but then the text says "if we run this it prints", when actually nothing will be printed without adding one more line to actually kick off the call to result in the print.

This is obviously just a nitpick, but just thought I'd point this out. The original issue I reported actually caught me off guard, but after that I knew to look for this so none of the other ones even registered with me until I thought about it later.
